x-pack/platform/plugins/shared/fleet/public/applications/fleet/sections/agents/agent_list_page/hooks/export_csv.tsx

```diff
@@ -70,7 +70,6 @@ export function useExportCSV() {
     const sort = getSortConfig(sortField, sortOrder) as EsQuerySortValue[];
 
     const searchSource: SearchSourceFields = {
-      type: 'search',
```

Comment:
I'm curious about what the intention of this `type` field is. what are valid values?

Reply:
It's not documented too well, I copied the logic originally from the reporting plugin. It seems something changed, I don't see `type` used anywhere else in the repo anymore.